wgpu-native/src/device.rs

```diff
@@ -518,6 +527,23 @@ impl Device<back::Backend> {
     }
 }
 
+impl<B: hal::Backend> fmt::Debug for Device<B> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
```

Comment:
is the queue group the only thing that isn't `Debug`? We might want an easier way to solve this

Reply:
Yes.